docs/platform/concepts/authentication-tokens.rst

```diff
@@ -0,0 +1,35 @@
+Authentication tokens
+=====================
+
+An authentication token allows a user to programmatically access resources instead of using their username-password. Unlike the username-password combination that has a longer shelf life (in some case, they never expire); you can limit how long an authentication token is valid for or revoke it completely.
+
+.. mermaid::
+
+sequenceDiagram
+    Client->>+Server: Here's my username/password. Give me a token.
+    Server-->>-Client: Your username/password is valid. Here's a token. 
+    Client->>+Server: Here's a token. Show me a list of my resources.
+    Server-->>-Client: Your token is valid. Here's a list of your resources.
+
+How Aiven manages user sessions using authentication tokens
+-----------------------------------------------------------
+
+Each time a user signs in to Aiven either via the web console, Aiven command line client, or direct REST API call, the server creates a new authentication token associated with the user.
+These tokens are set to expire after 30 days (subject to change) but the expiry date is adjusted whenever the token is used so the token may remain valid practically indefinitely if it is used frequently enough.
+
+Token counts
+------------
+
+The system has hard limits for how many valid authentication tokens are allowed per user. This limit is different for tokens that are created as a result of sign in operation and for tokens created explicitly; the limit for explicitly created tokens is small but the system never invalidates the tokens unless they expire or they are explicitly revoked. For automatically created tokens the limit is higher but when the limit is reached the system automatically deletes tokens that have been used least recently to avoid going above the limit.
```

Comment:
"small" and "higher" are not really quantitative information, and might generate more questions, do we have a number that we can share?

Reply:
Another good point. I have asked on an internal channel to get specific numbers.